src/org/omegat/gui/dictionaries/DictionariesTextArea.java

```diff
@@ -141,8 +150,9 @@ public void setFont(Font font) {
                 + " font-style: " + (font.getStyle() == Font.BOLD ? "bold"
                         : font.getStyle() == Font.ITALIC ? "italic" : "normal") + "; "
                 + " color: " + EditorColor.COLOR_FOREGROUND.toHex() + "; "
-                + " background: " + EditorColor.COLOR_BACKGROUND.toHex() + "; "
-                + " }");
+                + " background: " + EditorColor.COLOR_BACKGROUND.toHex() + ";} "
+                + ".word {font-size: " + (2 + font.getSize()) + "; font-style: bold; }"
```

Comment:
I don't see any effect from this change. In fact it looks like with this PR we are losing the effect of this stylesheet entirely. I guess it's because we are recreating the document every time. I think we need to re-set the stylesheet every time as well.